content/faq/bookmarks/how-to-import/index.fr.md

```diff
@@ -4,7 +4,7 @@ description: ""
 updated: "2024-06-12"
 
 taxonomies:
-  faq: ["Bookmarks"]
+  faq: ["bookmarks"]
```

Comment:
nit: Can the key contain spaces? "bookmarks and tracks"? Or dashes? Then it would be easier to translate properly (and a better SEO).

Reply:
Currently I use simple concatenation `"faq_section_" + taxonome_term` to get translation constant `faq_section_tts`. We can allow spaces and hyphens using Zola built-in [regex-replace](https://www.getzola.org/documentation/templates/overview/#regex-replace) function.

Reply:
So this key should also match the directory name where pages are located, right?